app/liquid/liquid_renderer.rb

```diff
@@ -43,6 +44,15 @@ def images
 
   private
 
+  def set_locale
+    begin
+      if @page.language.present? && @page.language.code.present?
+        I18n.locale = @page.language.code
+      end
+    rescue I18n::InvalidLocale
```

Comment:
If you're intentionally squashing this error, could you leave a comment to that effect?